src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -344,4 +345,24 @@ inline jobject NativeRayObjectToJavaNativeRayObject(
   return java_obj;
 }
 
+inline jobject NativeRayFunctionDescriptorToJavaStringList(
```

Comment:
We can just convert the C++ FD directly to Java FD, instead of having a temporary byte list.

Reply:
This will change a lot of code, I think PR should focus on Python support.